### DISCUSSION TITLE
Just 2D?

### DISCUSSION THREAD

**Discussion by larrywyre** [dyn4j/dyn4j#220] [Ideas]
The documentation says this is entirely a 2D model system. Have you thought about extending to 3D?

The reason I ask is that I am in the process of building a stellar plotting system using GAIA DR2 data that science fiction writers (the asked me to do it :) ) wish to use to visualize and experiment with for their stellar empires and plots.  
I have all the 3D graphics and plotting working. I am using OreKit for orbital dynamics and wanted to add a generalized physics engine for other things. 3D is required for my needs.

**Comment by wnbittle** [comment]
Nothing past a thought to be honest.  Nearly every algorithm would need to change, some in a significant way.  You'd be better off using a library that already exists.  It would take me way too long given how much free time I have these days.

**Comment by larrywyre** [comment]
I found your earlier answer, thanks for that.
I was looking around and found the toxiclibs project which does seem to have what I want.
Thanks and good luck
